zwave_js/rootfs/etc/cont-init.d/config.sh

```diff
@@ -145,13 +145,18 @@ else
     presets="$(printf '%s\n' "${presets_array[@]}" | jq -R . | jq -s .)"
 fi
 
+log_to_file=$(bashio::config "log_to_file")
+log_max_files=$(bashio::config "log_max_files")
+
 # Generate config
 bashio::var.json \
     s0_legacy "${s0_legacy}" \
     s2_access_control "${s2_access_control}" \
     s2_authenticated "${s2_authenticated}" \
     s2_unauthenticated "${s2_unauthenticated}" \
     log_level "${log_level}" \
+    log_to_file "${log_to_file}" \
```

Comment:
Is the log to file option in zwave-js compatible with the other log transports that we use? It sounds like it should be as the docs use the phrase "in addition":

https://zwave-js.github.io/node-zwave-js/#/api/driver?id=logconfig

Reply:
Ah I misread, yes let me update the docs good callout

Reply:
Have we confirmed that logging to file is compatible with the other log transports the server uses?

Reply:
I tested the UI functionality in HA and it works.

When we set `logToFile`, we do it when creating the driver. The server only starts after the driver has started, and when we need to use our own log transport, we push it into the transports list which would preserve the existing transports, which in this case is the file transport. So I don't see how the two could come in conflict

Reply:
Ok.